package.yaml

```diff
@@ -119,6 +119,8 @@ tests:
     - inspection-testing >= 0.4.2 && < 0.5
     - hspec >= 2.6.0 && < 3
     - doctest >= 0.16.0.1 && < 0.17
+    generated-other-modules:
+    - Build_doctests
```

Comment:
What does adding this here do?

Reply:
I was getting this warning for both packages:
```
Warning: The following modules should be added to exposed-modules or other-modules in /home/matej/Haskell/polysemy/polysemy.cabal:
             - In test:polysemy-test:
                 Build_doctests
             - In test:polysemy-test:
                 Build_doctests
         
         Missing modules in the cabal file are likely to cause undefined reference errors from the linker, along with other problems.
```
Should I split it? I find it to be a pretty small change.

Reply:
I don't mind, I was just wondering